pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -1185,7 +1185,8 @@ private void synchronize(
       final WebSocketConfiguration webSocketConfiguration,
       final MetricsConfiguration metricsConfiguration,
       final Optional<PermissioningConfiguration> permissioningConfiguration,
-      final Collection<EnodeURL> staticNodes) {
+      final Collection<EnodeURL> staticNodes)
+      throws IOException {
```

Comment:
I can't immediately see why this would now throw IOException.  I assume there's a chain of calls it now flows up that's not obvious in GitHub's UI but could you double check it is required with the final code?

Reply:
Removed - was hacked in, but resolved more suitably.